packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -145,6 +146,10 @@ Future<XcodeBuildResult> buildXcodeProject({
     return new XcodeBuildResult(success: false);
   }
 
+  String developmentTeam;
+  if (codesign && mode != BuildMode.release && buildForDevice)
```

Comment:
Will do in next PR. Need to pick a distribution certificate instead of a developer certificate. Need different fallback instructions etc.